spec/template.dd

```diff
@@ -391,9 +391,9 @@ immutable(S)
 )
 
     $(P This is especially useful when used with inheritance. For example,
-        you might want to implement a final base method which returns a derived
-        class type. Typically you would return a base type, but this won't allow
-        you to call or access derived properties of the type:)
+        consider implementing a final base method which returns a derived
+        class type. Typically this would return a base type, but this won't allow
```

Comment:
Replacing "you" with "this" here has rendered the "this" in "but this won't" redundant and awkward. The "won't" was already awkward anyway, as it doesn't jibe with the "would" in the preceding clause. Something about "not allow" feels off to me in this context also. I suggest:

"but this won't allow" -> "but that would prohibit"